mrp_product_document_link/views/mrp_production_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="mrp_production_form" model="ir.ui.view">
+        <field name="name">mrp.production.form (in mrp_product_document_link)</field>
+        <field name="inherit_id" ref="mrp.mrp_production_form_view"/>
+        <field name="model">mrp.production</field>
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='miscellaneous']" position="before">
+                <page string="Documents" name="documents">
```

Comment:
"Documents" is too generic, and also, is it possible to make this page invisible if there are no documents?
```suggestion
                <page string="Product Documents" name="product_documents">
```